### PR TITLE
19.46 update

### PR DESCRIPTION
**Pull request by pwozniak89**

### CODE REVIEW

File: lang/fr.UTF-8 [hunk 1]
```diff
@@ -1092,11 +1092,11 @@ theme_xhred_live_stats_disk=E/S disque
...
 theme_xhred_global_hours=Heures
 theme_xhred_title_locale_time_format=Format de l'heure
-theme_xhred_nice_size_PIB=PiB
-theme_xhred_nice_size_TIB=TiB
-theme_xhred_nice_size_GIB=GiB
-theme_xhred_nice_size_MIB=MiB
-theme_xhred_nice_size_kIB=KiB
+theme_xhred_nice_size_PIB=Po
+theme_xhred_nice_size_TIB=To
+theme_xhred_nice_size_GIB=Go
```
Comment (iliarostovtsev) [inline lang/fr.UTF-8]: KiB ne KB, MiB ne MB, GiB ne GB.

If in your case "Go" = GB or Mo = MB, then it's incorrect.

Comment (pwozniak89) [reply lang/fr.UTF-8]: Yes in French, Byte (B) is octet (o), so **B** must be replaced by **o**

Comment (iliarostovtsev) [reply lang/fr.UTF-8]: Those are Kibi-, Mebi-, Gibi- and etc.
https://blog.digilentinc.com/mib-vs-mb-whats-the-difference/

Comment (pwozniak89) [reply lang/fr.UTF-8]: Sorry... I'm wrong !
It must be Pio ; Tio ; Gio ; Mio ; kio

**Comment by iliarostovtsev** [comment]
Okay, merged and fixed manually. Thanks.

**Pull request merged**